mentors/cpp/n.json

```diff
@@ -1 +1,10 @@
-[]
+[
+  {
+    "github_username" : "ningke",
+    "name" : "Ning Ke",
+    "link_text" : "github profile",
```

Comment:
You might consider capitalizing github like GitHub

Reply:
Nice catch.  I agree.

Reply:
`Rakefile` isn't sensitive to `github`, just `Github`....  easily remedied....

Reply:
#791 created to guard against any form of miss-capitalized, but properly spelled, form of GitHub.

Reply:
Thanks @rpottsoh!